server/repositories/TransferRepository.js

```diff
@@ -123,6 +123,16 @@ class TransferRepository extends BaseRepository {
       )
       .where((builder) => this.whereBuilder(filter, builder));
 
+    if (limitOptions && limitOptions.sort_by) {
+      const column = limitOptions.sort_by.startsWith('-') ? 
+        limitOptions.sort_by.slice(1) : limitOptions.sort_by;
+      const order = limitOptions.sort_by.startsWith('-') ? 'desc' : 'asc';
```

Comment:
@XLPeng57 let's also introduce another query field that allows the client to specify the order (desc or asc). It should also have a default as well

Reply:
Sure. I'll add another query field for the order.